build(deps): bump http-cache-semantics from 4.1.0 to 4.1.1 (#1143)
redpanda-data/documentation: modules/reference/pages/rpk/rpk-cluster/rpk-cluster-config-lint.adoc

Remove any deprecated content from `redpanda.yaml`.

> Deprecated content includes properties that were set in the `redpanda.yaml` file. 

Reviewer: ```suggestion
Deprecated content includes properties that were set in the `redpanda.yaml` file 
```

Remove any deprecated content from `redpanda.yaml`.

> Deprecated content includes properties which were set via `redpanda.yaml`

Reviewer 2: ```suggestion
Deprecated content includes properties that were set in the `redpanda.yaml` file. 
```
